lib/vmpooler/pool_manager.rb

```diff
@@ -448,39 +460,111 @@ def _check_snapshot_queue
       unless vm.nil?
         begin
           vm_name, snapshot_name = vm.split(':')
-          revert_vm_snapshot(vm_name, snapshot_name)
+          revert_vm_snapshot(vm_name, snapshot_name, vsphere)
         rescue
           $logger.log('s', "[!] [snapshot_manager] snapshot revert appears to have failed")
         end
       end
     end
 
+    def find_vsphere_pool_vm(pool, vm, vsphere)
+      vsphere.find_vm(vm) || vsphere.find_vm_heavy(vm)[vm]
+    end
+
+    def migration_limit(migration_limit)
+      # Returns migration_limit setting when enabled
+      return false if migration_limit == 0 or not migration_limit
```

Comment:
In ruby the operator precedence of `||` vs. `or`, and `!` vs. `not` can be surprising.  I *think* this is ok here, but would prefer we use `||` and `!` instead of `or` and `not`.